package.json

```diff
@@ -109,11 +109,12 @@
     "prepublishOnly": "npm run build",
     "test": "react-scripts test --detectOpenHandles",
     "test:ci": "cross-env CI=true react-scripts test --passWithNoTests",
-    "lint": "eslint \"src/**/*.{js,jsx,ts,tsx}\"",
-    "lint:fix": "eslint \"src/**/*.{js,jsx,ts,tsx}\" --fix",
+    "lint": "eslint \"src/**/*.{js,jsx,ts,tsx}\" \"scripts/**/*.{js,ts}\"",
+    "lint:fix": "eslint \"src/**/*.{js,jsx,ts,tsx}\" \"scripts/**/*.{js,ts}\" --fix",
     "lint-staged": "lint-staged",
     "commitlint": "commitlint",
-    "coveralls": "npm run test:ci -- --coverage --watchAll=false && cat ./coverage/lcov.info"
+    "coveralls": "npm run test:ci -- --coverage --watchAll=false && cat ./coverage/lcov.info",
+    "translation:check": "ts-node-script scripts/checkMissingTranslations.ts"
```

Comment:
I think that we should run this script when `yarn start` and `yarn test` are run.

Reply:
Yes we can

Reply:
I don't like adding ts-node-script as new dependency. Can we manage this in some other way?

Reply:
@marcosvega91 any news about this?

Reply:
I asked more info about this in the other comment of @fox1t. I don't understand if he wants to keep .ts script and add a new command to convert to .js and run it or do something else

Reply:
@marcosvega91 I am really sorry I completely missed you response! I think we can add bin folder that contains compiled script and use node to run it. Basically add this file to the standard compilation done by our tsconfig.json. What do you think about it @marcosvega91 ?

Reply:
I have updated the script. Now ts file is compiled to js and then run as node script